Src/DDSTextureLoader.cpp

```diff
@@ -155,10 +155,20 @@ namespace
 
         HRESULT hr = E_FAIL;
 
-        if (forceSRGB)
+        uint32_t exclusiveFlags = DDS_LOADER_FORCE_SRGB | DDS_LOADER_IGNORE_SRGB | DDS_LOADER_SRGB_DEFAULT;
```

Comment:
Generally I don't worry about "extra" flags in the implementation. "Unknown" bits are just ignored to avoid having to maintain these checks. Also, since we are using `DEFINE_ENUM_FLAG_OPERATORS` there's already a level of type-safety, so this check is not needed since it's less likely you'll mix up enum sets.

Reply:
Fair enough.  In this particular case it's not really checking for "unknown" bits, but rather them explicitly specifying two flags that are in direct contradiction to each other.  But, it's also fine to say "undefined behavior" in that case if maintaining the check is outside the normal parameter validation methodology in the project.